Formula/onedpl.rb

```diff
@@ -0,0 +1,36 @@
+class Onedpl < Formula
+  desc "C++ standard library algorithms with support for execution policies"
+  homepage "https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/dpc-library.html"
+  url "https://github.com/oneapi-src/oneDPL/archive/refs/tags/oneDPL-2021.4.0-release.tar.gz"
+  sha256 "74ffc0fabd6a74af8c1a7a92f049794559bbe5afcd755398f1b130226f1b60fa"
+  # Apache License Version 2.0 with LLVM exceptions
+  license "Apache-2.0" => { with: "LLVM-exception" }
+
+  depends_on "tbb"
+
+  def install
+    prefix.install "include"
```

Comment:
Can we use their CMake-based installer so that this installs the CMake config files too? We might need to pass `-DONEDPL_ENABLE_SIMD=OFF` on ARM, though.

Reply:
Unfortunately CMake-based installer installs nothing in this version, this needs https://github.com/oneapi-src/oneDPL/commit/008dd241e4766bda87e439ebe64a0f91d795b181

Reply:
Does that commit apply cleanly? If so, maybe we should use a patch. Or at least, can we add a comment that says to use CMake to install when that commit lands in a release?